### PR TITLE
gitignore node diagnostic reports

### PR DESCRIPTION
**Pull request by erikdstock**
#minor

This PR adds a line to gitignore node diagnostic reports. It is copied from https://gitignore.io/api/node.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,6 @@ yalc.lock
 coverage*
 reports*
 2
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
```
Comment (zephraph) [inline .gitignore]: Could this just be `report.*.json`?

Comment (erikdstock) [reply .gitignore]: I used this because it was the 'official' line from gitignore.io. Don't have a strong opinion either way if you think that is better.

Comment (erikdstock) [reply .gitignore]:
```suggestion
report.*.json
```

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.38.0` :rocket:

**Pull request merged**